src/step_definitions/performance.js

```diff
@@ -43,7 +46,24 @@ defineSupportCode(function ({ When }) {
     });
 
     return browser.driver.wait(() => {
-      return proxyDone;
+      return (
+        this.performanceReportFile = uniqueFileName,
+        proxyDone
+      )
     });
   });
+
+  Then(/^the requests should take a maximum of "([^"]*)" milliseconds$/, function (maxTiming) {
+    const slowRequests = analyser.checkTiming(this.performanceReportFile, maxTiming);
```

Comment:
the `maxTiming` is a number or string? If string then I guess you may have problems here